inyoka/wiki/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -676,3 +673,6 @@ msgstr "Seiten"
 msgid "“%(title)s” was reverted successfully"
 msgstr "Die Seite „%(title)s“ wurde erfolgreich wiederhergestellt."
 
+#~ msgid "Expandable Articles"
+#~ msgstr "Ausbaufähige Artikel"
+
```

Comment:
as you deleted the only occurrence in the HTML, you can also delete the commented translation (in the worst case we have the git log)

```suggestion

```